src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -410,77 +426,92 @@ def HandleCommissioningComplete(nodeId: int, err: PyChipError):
             else:
                 LOGGER.warning("Failed to commission: {}".format(err))
 
-            self._dmLib.pychip_DeviceController_SetIcdRegistrationParameters(False, None)
+            self._dmLib.pychip_DeviceController_SetIcdRegistrationParameters(
+                False, None)
 
             if self._dmLib.pychip_TestCommissionerUsed():
                 err = self._dmLib.pychip_GetCompletionError()
 
             if self._commissioning_context.future is None:
-                LOGGER.exception("HandleCommissioningComplete called unexpectedly")
+                LOGGER.exception(
+                    "HandleCommissioningComplete called unexpectedly")
```

Comment:
Not sure what reformatted this, but we use line length of 132 by default. So this (and many other lines) should not wrap.

Reply:
that was VSCode, I was using auopep8 without the required settings: created a PR to resolve that here: #34475